argocd/tests/fixtures/api_server_metrics.txt

```diff
@@ -0,0 +1,1712 @@
+# HELP argocd_redis_request_duration Redis requests duration.
```

Comment:
How does a file like this get generated?

Reply:
I think it's a normally used format these days:
https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md

From my understanding vendors such as ArgoCD and others create their own exporters that export metrics in this format and the agent is able to read and parse this format